metals/src/main/scala/scala/meta/internal/metals/BloopServers.scala

```diff
@@ -29,6 +31,25 @@ final class BloopServers(
     client: MetalsBuildClient
 )(implicit ec: ExecutionContextExecutorService) {
 
+  def shutdownServer(): Boolean = {
+    val dummyIn = new ByteArrayInputStream(new Array(0))
+    val cli = new BloopgunCli(
+      BuildInfo.bloopVersion,
+      dummyIn,
+      System.out,
+      System.err,
+      Shell.default
+    )
+    val result = cli.run(Array("ng-stop")) == 0
```

Comment:
What is the difference between ngstop and exit?

Reply:
I think that is the underlying nailgun command, might be better to use exit. I was testing it previously and it didn't seem to work with exit, but that might have been a different thing.

Reply:
Switched to `exit` since it works with it and that is a Bloop command.